src/main/scala/scalismo/faces/image/PixelImageNormalization.scala

```diff
@@ -23,18 +23,20 @@ object PixelImageNormalization {
 
   /** Normalizes the image such that its values are on the interval [lower, upper]. */
   def normalizeDoubleImageToRange(image: PixelImage[Double], lower: Double, upper: Double): PixelImage[Double] = {
-    def normalizer(x: Double) = (x - lower) / (upper - lower)
-    if (lower < 0.0 || upper > 1.0)
-      image
-    else
+    require(lower < upper)
+    val min = image.values.min
+    val max = image.values.max
+    if(min == max){ //if we would divide with 0
+      image.map(_ => (upper-lower)/2) //return mean of [lower,upper]
```

Comment:
Here it should be `(lower+upper)/2` or `lower+(upper-lower)/2`. I would calculate the constant one above so that it is even better visible that the map does not depend on the position.